webpack.config.dev.js

```diff
@@ -38,7 +38,8 @@ export default {
     }),
     new webpack.DefinePlugin({
       'teencode.feature': featureFlags,
-      'process.env.FB_APPID': JSON.stringify(process.env.FB_APPID)
+      'process.env.FB_APPID': JSON.stringify(process.env.FB_APPID),
+      'process.env.DISQUS_SHORT_NAME': JSON.stringify(process.env.DISQUS_APP_NAME)
```

Comment:
Could we extract the env variables definition into a separate GLOBALS constant? As we have it [here in prod config](https://github.com/teencode-org/teencode/blob/61917b12bcfaae19cd11dda447bdb6e4afb059bd/webpack.config.prod.js#L10)